snmp/assets/configuration/spec.yaml

```diff
@@ -298,21 +298,25 @@ files:
       - name: discovery_interval
         description: |
           Interval between 2 discovery starts, in seconds.
-          Only available using python SNMP integration.
         value:
           type: integer
           example: 3600
       - name: discovery_allowed_failures
         description: |
           Number of times a discovered host can fail before it is removed from the discovered list.
-          Only available using python SNMP integration.
         value:
           type: integer
           example: 3
       - name: workers
         description: |
-          Number of workers used for check when using discovery.
-          Only available using python SNMP integration.
+          Number of workers used for check collection (metrics, service checks, metadata, etc) when using discovery.
+        value:
+          type: integer
+          example: 5
+      - name: discovery_workers
```

Comment:
What is the use case for `discovery_workers` != `workers`? Wonder if we should use only `workers` also for discovery threads

Reply:
I think it's good to have separate configuration: autodiscovery and check probably do not need the same number of workers (likely more workers for checks than for AD, btw)

Reply:
The `discovery_workers` here is similar to `snmp_listener.workers`.

I think it's helpful to be able to tweak # of workers used to discover (how many devices to discover concurrently) separate from # of workers used to run checks (how many devices to run concurrently).

That said, since this implementation (SNMP corecheck Autodiscovery) is mainly for backward compat, I don't see many users tweaking `discovery_workers`. If they have large subnets, it's better to use snmp_listener anyway.